writing-style-guide.md

```diff
@@ -0,0 +1,267 @@
+# Writing Style Guide
+
+> **DRAFT**
```

Comment:
Personally I would mark the PR as draft; but if we intend that this advice is no longer draft when we merge it, then I wouldn't have this text here.

Reply:
Removed.